docs/setup/index.rst

```diff
@@ -57,8 +67,19 @@ is equivalent to running the following sequence of commands:
     > pip install rastervision_core=={{ version }}
     > pip install rastervision_pytorch_learner=={{ version }}
     > pip install rastervision_pytorch_backend=={{ version }}
+
+Another optional plugin that is available, but not installed by default,  is :mod:`rastervision.gdal_vsi`.
+
+.. code-block:: console
+
     > pip install rastervision_gdal_vsi=={{ version }}
 
+The command above will attempt to install GDAL via ``pip``. If that fails, you can instead try installing via ``conda`` as shown below. Replace ``<version>`` with the version listed `here <{{ repo }}/rastervision_gdal_vsi/requirements.txt>`__.
```

Comment:
This shouldn't be necessary on Linux because `rasterio` now installs the GDAL binary and Python package.

Reply:
It doesn't seem to install the python package for me:

```
$ pip install rasterio==1.3.2
Collecting rasterio==1.3.2
  Using cached rasterio-1.3.2-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (20.9 MB)
Requirement already satisfied: snuggs>=1.4.1 in ./anaconda3/lib/python3.9/site-packages (from rasterio==1.3.2) (1.4.7)
Requirement already satisfied: attrs in ./anaconda3/lib/python3.9/site-packages (from rasterio==1.3.2) (21.4.0)
Requirement already satisfied: certifi in ./anaconda3/lib/python3.9/site-packages (from rasterio==1.3.2) (2022.9.14)
Requirement already satisfied: cligj>=0.5 in ./anaconda3/lib/python3.9/site-packages (from rasterio==1.3.2) (0.7.2)
Requirement already satisfied: numpy>=1.18 in ./anaconda3/lib/python3.9/site-packages (from rasterio==1.3.2) (1.23.3)
Requirement already satisfied: click-plugins in ./anaconda3/lib/python3.9/site-packages (from rasterio==1.3.2) (1.1.1)
Requirement already satisfied: affine in ./anaconda3/lib/python3.9/site-packages (from rasterio==1.3.2) (2.3.1)
Requirement already satisfied: setuptools in ./anaconda3/lib/python3.9/site-packages (from rasterio==1.3.2) (63.4.1)
Requirement already satisfied: click>=4.0 in ./anaconda3/lib/python3.9/site-packages (from rasterio==1.3.2) (8.1.3)
Requirement already satisfied: pyparsing>=2.1.6 in ./anaconda3/lib/python3.9/site-packages (from snuggs>=1.4.1->rasterio==1.3.2) (3.0.9)
Installing collected packages: rasterio
Successfully installed rasterio-1.3.2
```

And
```
$ pip freeze | grep gdal
```
finds nothing.

Reply:
I was wrong.`rasterio` includes the GDAL binary, but doesn't install the `gdal` python package.